doc/mathdoc/equations.rst

```diff
@@ -802,6 +824,90 @@ In script ``urbs.py`` the constraint sell buy symmetry rule is defined and calcu
 		else:
 			return pyomo.Constraint.Skip
 
+**Process Throughput by Partial Rules**: These constraint process throughput by partial rules constrict the process throughput :math:`\tau_{vpt}` either between the total process capacity :math:`\kappa_{vp}` and its minimum allowable partial load (:math:`\kappa_{vp}` * :math:`\underline{P}_{vp}`), or zero. 
+
+In script ``urbs.py`` the constraint process throughput by partial rules are defined and calculated by the following code fragment:
+::
+
+    m.res_process_throughput_by_partial_1 = pyomo.Constraint(
```

Comment:
Here and at the other partial/startup rules, the `.. math::` version of the constraints is missing.